core/blockchain_test.go

```diff
@@ -3238,3 +3238,113 @@ func TestTransactionCountLimit(t *testing.T) {
 		t.Fatalf("error mismatch: have: %v, want: %v", err, consensus.ErrInvalidTxCount)
 	}
 }
+
+func TestEIP3651(t *testing.T) {
+	var (
+		aa     = common.HexToAddress("0x000000000000000000000000000000000000aaaa")
+		bb     = common.HexToAddress("0x000000000000000000000000000000000000bbbb")
+		engine = ethash.NewFaker()
+		db     = rawdb.NewMemoryDatabase()
+
+		// A sender who makes transactions, has some funds
+		key1, _ = crypto.HexToECDSA("b71c71a67e1177ad4e901695e1b4b9ee17ae16c6668d313eac2f96dbcda3f291")
```

Comment:
as for unit test. i think we need check the error

Reply:
I think there cannot be error here, because `key` that is parsed always same and valid. And also, checking error will complicate initialization block

Reply:
This conversion is not part of what we want to test here, just initialization, so I think it's fine to not check the error.

Reply:
ok